module/VuFind/src/VuFind/Http/GuzzleService.php

```diff
@@ -90,6 +89,19 @@ public function __construct(array $config)
         }
     }
 
+    /**
+     * Return a generic PSR-compliant client.
+     *
+     * @param ?string $url     Target URL (required for proper proxy setup for non-local addresses)
+     * @param ?float  $timeout Request timeout in seconds (overrides configuration)
+     *
+     * @return ClientInterface
+     */
+    public function createClient(?string $url = null, ?float $timeout = null): \Psr\Http\Client\ClientInterface
```

Comment:
This can be simplified for consistency with the comment, leveraging the use statement above:

```suggestion
    public function createClient(?string $url = null, ?float $timeout = null): ClientInterface
```